### PR TITLE
feat: Expose tracked screen frames

### PR DESCRIPTION
**Pull request by philipphofmann**
## :scroll: Description

Expose a method for HybridSDKs to get the current slow, frozen and total frames.

## :bulb: Motivation and Context

Hybrid SDKs want to add the frame rates to transactions.

## :green_heart: How did you test it?
CI and unit tests.

## :pencil: Checklist

- [x] I reviewed the submitted code
- [x] I added tests to verify the changes
- [x] I updated the docs if needed
- [x] Review from the native team if needed
- [x] No breaking changes

### CODE REVIEW

File: Tests/SentryTests/Integrations/TestDisplayLinkWrapper.swift [hunk 1]
```diff
@@ -25,6 +25,28 @@ class TestDiplayLinkWrapper: SentryDisplayLinkWrapper {
         target = nil
         selector = nil
     }
+    
+    func givenFrames(_ slow: Int, _ frozen: Int, _ normal: Int) {
```
Comment (philipphofmann) [inline Tests/SentryTests/Integrations/TestDisplayLinkWrapper.swift:29]: I moved that here from SentryTracerTests. FYI

**Review by bruno-garcia — APPROVED** [review]
👍

**Review by jennmueng — APPROVED** [review]
Just got the chance to test it out with https://github.com/getsentry/sentry-react-native/pull/1711. Works great!

Would it be possible for a method to disable the frames tracking? I say method and not a flag passed to options because we're most likely going to put the option to disable and enable specific performance features in the `ReactNativeTracing` integration options instead of the sdk options.

**Pull request merged**